app/konnect/org-management/azure-idp.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Set up SSO with Azure
+content_type: how-to
+---
+
+As an alternative to Kong Konnect’s native authentication, you can set up single sign-on (SSO) access to Konnect through an identity provider (IdP) with OpenID Connect. This authentication method allows your users to log in to Kong Konnect using their IdP credentials, without needing a separate login.
+
+You can’t mix authentication methods in Kong Konnect. With IdP authentication enabled, all non-admin Konnect users have to log in through your IdP. Only the Konnect org owner can continue to log in with Konnect’s native authentication.
+
+{:.note}
+> This guide shows you how to configure SSO for Konnect with Microsoft Azure. See [Configure Azure IdP for Dev Portal](/konnect/dev-portal/access-and-approval/azure/) for those instructions.
+
+
+## Prerequisites
+### Create an application in Azure
+
+1. In [Azure](https://portal.azure.com/), navigate to **App registrations**. 
+
+1. Click **New registration** to register a new application:
+
+1. Name the application.
+
+1. Select **Accounts in this organizational directory only** for the **Supported account type**. 
+
+1. Select **Web** and enter the (ADD CORRECT URL FOR KONNECT HERE)
+    
+1. Save the application ID for later.
+
+
+### Configure the Azure application
+
+1. Click **New client secret**, enter a description, select an expiration value, and click **Add**.
+    
+    * Save the secret value for configuring {{site.konnect_short_name}}.
+
+
+1. Click **Overview** in the sidebar, then click the **Endpoints** tab.
+
+1. Copy the **OpenID Connect metadata document** URL and open it in your browser:
```

Comment:
question: i'm not sure if it's helpful, or simpler for end-users, but we can accept the discovery URL (**OpenID Connect metadata document** value) directly in the issuer form field.

Reply:
I think I'd prefer having more steps over potentially confusing users, but we can update it with the issuer form field instead if you think that would be better.